app/mailers/application_mailer.rb

```diff
@@ -38,11 +38,11 @@ def comment_notify(data, recipient)
     @message += comment_text(@data, key)
 
     if comment_on_moment(@data) || comment_on_moment_private(@data)
```

Comment:
There is a lot of duplication in here.
What if there was a method that returned the class name?

```
url = send("#{commented_model(@data)}_url", @data['typeid'])
link = link_to(I18n.t('click_here'), url)
```

```
def commented_model(data)
  data['type'].match(/comment_on_([^_]+)/)[1]
end
```